Mixpanel-swift.podspec

```diff
@@ -39,7 +39,7 @@ Pod::Spec.new do |s|
 
   s.subspec 'Complete' do |ss|
     ss.ios.pod_target_xcconfig = {
-      'OTHER_SWIFT_FLAGS' => '$(inherited) -D DECIDE'
+      'OTHER_SWIFT_FLAGS' => '$(inherited) -D AUTOMATIC_EVENTS'
```

Comment:
can we remove this flag and only reply on the setting from `init`?

Reply:
replaced with `#if os(iOS) || os(tvOS)`